packages/web-app-files/src/components/Search/List.vue

```diff
@@ -272,11 +272,13 @@ export default defineComponent({
     const tagParam = useRouteQuery('q_tags')
     const lastModifiedParam = useRouteQuery('q_lastModified')
     const mediaTypeParam = useRouteQuery('q_mediaType')
-    const fullTextParam = useRouteQuery('q_fullText')
+    const titleOnlyParam = useRouteQuery('q_titleOnly')
+
+    const fullTextSearchEnabled = computed(() => capabilityStore.searchContent?.enabled)
```

Comment:
Why not using storeToRefs for that ?

Reply:
Because it's nested in the capabilities, meaning even if we'd use `storeToRefs`, we would still need to further narrow it down via a `computed`.